### ISSUE TITLE
Histogram to Raw video 

### ISSUE THREAD

**Issue by BrainFartArt** [mirsadm/motioncam#59]
Histogram would be ideal for Raw Video.

**Comment by 2blackbar** [comment]
Yes i agree, , actual raw data histogram where dynamic range isnt destroyed by rec.709 in codecs.

**Comment by mirsadm** [comment]
I'm already generating the histogram from the RAW data, it just needs to be displayed. Something I'll do when I get a chance to do a clipping indicator

**Comment by 2blackbar** [comment]
Histogram would be great, on screen its hard to tell where most information is about darker areas, it would help to decide about over/underexposing particular scenes.
